docs/notes/previous_versions/v2.8.2.rst

```diff
@@ -3,42 +3,42 @@ Version 2.8.2 (EOL)
 
 This release includes the following **improvements**:
 
-1. Add ASAN CI tests for Fast DDS and Discovery Server 
-2. Mirror master onto both 2.8.x & 2.9.x 
-3. Doxygen documentation: add deprecation notice to `ThroughputControllerDescriptor` 
-4. Several fixes to remove warnings in Ubuntu Jammy (22.04) 
-5. Improve behavior when `HAVE_STRICT_REALTIME` is not set 
-6. Using functors in `StatefulWriter.cpp` for_matched_readers 
-7. Fix build on old compilers 
-8. Avoid creation of DynamicTypes on example 
-9. Implement a validity check for firstSN 
+1. Add ASAN CI tests for Fast DDS and Discovery Server
+2. Mirror master onto both 2.8.x & 2.9.x
+3. Doxygen documentation: add deprecation notice to `ThroughputControllerDescriptor`
+4. Several fixes to remove warnings in Ubuntu Jammy (22.04)
+5. Improve behavior when `HAVE_STRICT_REALTIME` is not set
+6. Using functors in `StatefulWriter.cpp` for_matched_readers
+7. Fix build on old compilers
+8. Avoid creation of DynamicTypes on example
+9. Implement a validity check for `firstSN`
 
 This release includes the following **bugfixes**:
 
-1. Fix bug in Topic creation with differfent Type Name 
-2. Fix tests failinf with subprocess aborted error 
-3. Add ASAN CI tests for Fast DDS and Discovery Server 
-4. Added `ignore_participant_flags()` to `Blackbox_FastRTPS` `PubSubReader`. 
+1. Fix bug in Topic creation with different Type Name
+2. Fix tests failing with subprocess aborted error
+3. Add ASAN CI tests for Fast DDS and Discovery Server
```

Comment:
`Add ASAN..` is already in the *improvements* section. I think that  following one was left
```suggestion
3. Fix communication with asymmetric ignoreParticipantFlags
```